src/dynamics/fixed_points/topology.lean

```diff
@@ -0,0 +1,41 @@
+/-
+Copyright (c) 2020 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov, Johannes Hölzl
+-/
+import dynamics.fixed_points.basic
+import topology.separation
+
+/-!
+# Topological properties of fixed points
+
+Currently this file contains two lemmas:
+
+- `is_fixed_pt_of_tendsto_iterate`: if `f^n(x) → y` and `f` is continuous at `y`, then `f y = y`;
+- `is_closed_fixed_points`: the set of fixed points of a continuous map is a closed set.
+
+## TODO
+
+fixed points, iterates
+-/
+
+variables {α : Type*} [topological_space α] [t2_space α] {f : α → α}
+
+open function filter
+open_locale topological_space
+
+/-- If the iterates `f^[n] x₀` converge to `x` and `f` is continuous at `x`,
+then `x` is a fixed point for `f`. -/
+lemma is_fixed_pt_of_tendsto_iterate {x : α} (hf : continuous_at f x)
+  (hx : ∃ x₀ : α, tendsto (λ n, f^[n] x₀) at_top (𝓝 x)) :
```

Comment:
The formulation of this lemma is strange: it would make more sense to remove the existential quantifier and have two variables `{x x₀}` (I know this is a preexisting formulation, but there is always room for improvement when moving things around :)